api/components/v1alpha1/modelregistry_types.odh.go

```diff
@@ -0,0 +1,28 @@
+//go:build !rhoai
+
+/*
+Copyright 2023.
```

Comment:
We should use 2025

```suggestion
Copyright 2025.
```

Also in other new files